dev-util/hip/hip-5.1.3-r1.ebuild

```diff
@@ -81,12 +81,14 @@ src_prepare() {
 
 	# correct libs and cmake install dir
 	sed -e "/LIB_INSTALL_DIR/s:PREFIX}/lib:PREFIX}/$(get_libdir):" \
-		-e "/${HIP_COMMON_DIR}\/cmake DESTINATION/s: .): $(get_libdir)):" -i CMakeLists.txt || die
+		-e "/\${HIP_COMMON_DIR}/s:cmake DESTINATION .):cmake/ DESTINATION share/cmake/Modules):" -i CMakeLists.txt || die
 	sed -e "/LIBRARY DESTINATION/s:lib:$(get_libdir):" -i src/CMakeLists.txt || die
 
 	sed -e "/\.hip/d" \
 		-e "s,DESTINATION lib,DESTINATION $(get_libdir),g" \
-		-e "/\(cmake\|samples\)/s,DESTINATION \.,DESTINATION share,g" \
+		-e "/samples/s,DESTINATION \.,DESTINATION share/hip,g" \
```

Comment:
The samples belong to `/usr/share/doc/${P}/`.

Reply:
Actually it is `/usr/share/doc/${PF}/`